markdown/exploring-solace-native-apis/exploring-solace-native-apis.md

```diff
@@ -148,31 +148,22 @@ Pick "Solace Messaging" and make a note of the username, password, Message VPN a
 
 ![Make a note of the username, password, Message VPN and SMF Host](./img/cloud-connection-details.png)
 
-Then use these details on the sdkperf command line: use the -cu ("client username") and -cp ("client password") options, so it looks something like:
+You'll need these details later.  But to connect Try-Me, just navigate to the Try-Me tab on the left hand menu:
 
-	sdkperf_java.sh -cip <SMF Host> -cu <username>@<message-vpn> -cp <password> -stl codelab/sample/topic
+![](./img/try-me.png)
 
-Once sdkperf starts, you should see something along the lines of:
+Now we need to connect.  The connection details are filled in, which makes things easy, so just go to the **Subscriber** and click the "Connect" button.
 
-	JAVA: /usr/bin/java
-	SOLACE_VM_ARGS: -Xms512m -Xmx1024m
-	Run Info: CPU usage currently disabled.
-	Client naming used:
-		logging ID   = perf_client000001
-		username     = perf_client000001
-		vpn          = 
-		client names = sdk generated.
-	
-	> VM Name: Java HotSpot(TM) 64-Bit Server VM
-	Run Info: Using Java Nanosecond Timer for Timing
-	> Timing Package Clock Speed (Hz): 1000000000
-	> Getting ready to init clients
-	> Adding subscriptions if required
-	> Getting ready to start clients.
-	PUB MR(5s)=    0, SUB MR(5s)=    0, CPU=0
-	PUB MR(5s)=    0, SUB MR(5s)=    0, CPU=0
+Positive
+:  If you're using Docker, you may not connect.  This is because the default docker compose yaml doesn't expose the port Try-Me uses.  Simply stop the containerand edit the .yml file.  Fine the Port Mappings section, and uncomment the line below "#Web transport"
```

Comment:
Typo? 
_Find_ the Port...

Reply:
I see what you mean, but no, in this instance _expose_ is the right phrase.  You tell Docker which ports on your container you want people to be able to access - you expose the port.